internal/getproviders/package_authentication.go

```diff
@@ -41,12 +42,25 @@ func (t *PackageAuthenticationResult) String() string {
 	}
 	return []string{
 		"verified checksum",
-		"official provider",
-		"partner provider",
-		"community provider",
+		"signed by HashiCorp",
+		"signed by HashiCorp partner",
+		"self-signed",
```

Comment:
I like the changes here, but we've been using the official/partner/community terminology based on internal RFCs. Is there an RFC or other formal approval that indicates we should switch to this new phrasing? If not, can we get something in writing before the release so we can start updating it elsewhere?